src/api/api.js

```diff
@@ -184,8 +184,8 @@ class PugApi {
   /** get invites for user */
 
   static async getInvites(username) {
-    let res = await this.request(`users/${username}/invites}`);
-    return res.invtes;
+    let res = await this.request(`users/${username}/invites`);
```

Comment:
this is a `const`

Reply:
actually it looks like most of these `res` variables are const. Not a big deal just being nitpicky